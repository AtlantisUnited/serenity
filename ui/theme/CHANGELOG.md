# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 0.0.5 (2021-05-10)

**Note:** Version bump only for package @ui/theme

## 0.0.4 (2020-09-28)

### Bug Fixes

- **accounts:** add emotion-server, delete font ([08acf5a](https://github.com/Atlantis-Lab/serenity/commit/08acf5a7169352a606925fdfbbaa18f9917a8106))

## 0.0.3 (2020-06-30)

**Note:** Version bump only for package @ui/theme

## 0.0.2 (2020-06-10)

**Note:** Version bump only for package @ui/theme
