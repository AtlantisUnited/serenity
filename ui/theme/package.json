{
  "name": "@ui/theme",
  "version": "0.0.5",
  "license": "MIT",
  "main": "src/index.ts",
  "devDependencies": {
    "@types/react": "^17.0.6",
    "@types/styled-system": "5.1.4"
  },
  "dependencies": {
    "@emotion/css": "^11.1.3",
    "@emotion/react": "^11.4.0",
    "@emotion/styled": "^11.3.0",
    "react": "^17.0.2",
    "styled-system": "^5.1.0"
  }
}
