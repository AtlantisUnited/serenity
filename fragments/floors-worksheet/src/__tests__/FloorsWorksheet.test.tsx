import React                     from 'react'
import { IntlProvider, useIntl } from 'react-intl'

import { ThemeProvider }         from '@ui/theme'
import { render }                from '@testing-library/react'

import { FloorsWorksheet }       from '../FloorsWorksheet'

const TestComponent = () => {
  const intl = useIntl()

  return <FloorsWorksheet intl={intl} />
}

describe('Floors worksheet', () => {
  describe('snapshots', () => {
    it('should match latest render snapshot', () => {
      const { asFragment } = render(
        <IntlProvider locale='ru' defaultLocale='ru' messages={{}}>
          <ThemeProvider>
            <TestComponent />
          </ThemeProvider>
        </IntlProvider>
      )

      expect(asFragment()).toMatchSnapshot()
    })
  })
})
