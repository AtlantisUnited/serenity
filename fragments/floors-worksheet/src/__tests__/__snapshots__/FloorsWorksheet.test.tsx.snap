// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Floors worksheet snapshots should match latest render snapshot 1`] = `
<DocumentFragment>
  .emotion-143 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-8 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding-left: 30px;
  padding-right: 30px;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  box-sizing: border-box;
}

.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-basis: 20px;
  -ms-flex-preferred-size: 20px;
  flex-basis: 20px;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-1 {
  font-weight: 500;
  line-height: 1.6;
  font-family: Inter;
  font-size: 13px;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-basis: 16px;
  -ms-flex-preferred-size: 16px;
  flex-basis: 16px;
}

.emotion-7 {
  max-width: 180px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-6 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-4 {
  box-sizing: border-box;
  outline: none;
  border: none;
  width: 100%;
  font-family: Inter;
  font-weight: 500;
  border-radius: 4px;
  border: 1px solid rgba(0,0,0,0.08);
  color: rgb(0,0,0);
  background-color: rgb(255,255,255);
  font-size: 12px;
  height: 50px;
  padding-left: 20px;
  padding-right: 20px;
  border-top-right-radius: 0;
  border-bottom-right-radius: 0;
}

.emotion-4::-webkit-input-placeholder {
  color: rgb(190,190,190);
}

.emotion-4::-moz-placeholder {
  color: rgb(190,190,190);
}

.emotion-4:-ms-input-placeholder {
  color: rgb(190,190,190);
}

.emotion-4::placeholder {
  color: rgb(190,190,190);
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  box-sizing: border-box;
  font-family: Inter;
  font-weight: 500;
  border-radius: 4px;
  border: 1px solid rgba(0,0,0,0.08);
  color: rgb(0,0,0);
  background-color: rgb(255,255,255);
  font-size: 12px;
  padding-right: 20px;
  padding-left: 20px;
  height: 50px;
  border-bottom-left-radius: 0;
  border-top-left-radius: 0;
  border-left-width: 0;
}

.emotion-9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-basis: 30px;
  -ms-flex-preferred-size: 30px;
  flex-basis: 30px;
}

.emotion-10 {
  width: 100%;
  height: 1px;
  background: rgb(235,235,235);
}

.emotion-11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-basis: 22px;
  -ms-flex-preferred-size: 22px;
  flex-basis: 22px;
}

.emotion-24 {
  box-sizing: border-box;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

@media screen and (min-width:40em) {
  .emotion-24 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
  }
}

.emotion-18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-flex-basis: 100%;
  -ms-flex-preferred-size: 100%;
  flex-basis: 100%;
}

@media screen and (min-width:40em) {
  .emotion-18 {
    -webkit-flex-basis: 100%;
    -ms-flex-preferred-size: 100%;
    flex-basis: 100%;
  }
}

@media screen and (min-width:52em) {
  .emotion-18 {
    -webkit-flex-basis: 50%;
    -ms-flex-preferred-size: 50%;
    flex-basis: 50%;
  }
}

.emotion-17 {
  font-family: Inter;
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  white-space: nowrap;
  cursor: pointer;
  border: none;
  outline: none;
  padding: 0;
  width: 100%;
  font-size: 12px;
  padding: 0px 16px;
  height: 50px;
  border: 1px solid rgba(0,0,0,0.08);
  color: rgb(158,158,158);
  background: rgb(255,255,255);
  border-radius: 4px;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
}

.emotion-17:active {
  -webkit-transform: scale(0.98);
  -ms-transform: scale(0.98);
  transform: scale(0.98);
}

.emotion-15 {
  margin-right: 12px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-16 {
  white-space: normal;
  color: rgb(153,153,153);
  font-size: 12px;
  font-weight: 600;
  line-height: 1.3;
  text-align: left;
  font-family: Inter;
}

.emotion-19 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-basis: 22px;
  -ms-flex-preferred-size: 22px;
  flex-basis: 22px;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
}

.emotion-109 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-basis: 0;
  -ms-flex-preferred-size: 0;
  flex-basis: 0;
}

@media screen and (min-width:40em) {
  .emotion-109 {
    -webkit-flex-basis: 0;
    -ms-flex-preferred-size: 0;
    flex-basis: 0;
  }
}

@media screen and (min-width:52em) {
  .emotion-109 {
    -webkit-flex-basis: 22px;
    -ms-flex-preferred-size: 22px;
    flex-basis: 22px;
  }
}

.emotion-110 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
  -webkit-flex-basis: 0;
  -ms-flex-preferred-size: 0;
  flex-basis: 0;
}

@media screen and (min-width:40em) {
  .emotion-110 {
    -webkit-box-flex: 0;
    -webkit-flex-grow: 0;
    -ms-flex-positive: 0;
    flex-grow: 0;
    -webkit-flex-basis: 0;
    -ms-flex-preferred-size: 0;
    flex-basis: 0;
  }
}

@media screen and (min-width:52em) {
  .emotion-110 {
    -webkit-box-flex: 1;
    -webkit-flex-grow: 1;
    -ms-flex-positive: 1;
    flex-grow: 1;
    -webkit-flex-basis: 50%;
    -ms-flex-preferred-size: 50%;
    flex-basis: 50%;
  }
}

<div
    class="emotion-143"
    display="flex"
    width="100%"
  >
    <div
      class="emotion-8"
      display="flex"
      width="100%"
    >
      <div
        class="emotion-0"
        display="flex"
      />
      <div
        class="emotion-2"
        display="flex"
      >
        <span
          class="emotion-1"
          font-family="primary"
          font-size="normal"
          font-weight="medium"
        >
          Площадь пола?
        </span>
      </div>
      <div
        class="emotion-3"
        display="flex"
      />
      <div
        class="emotion-7"
        display="flex"
      >
        <div
          class="emotion-6"
        >
          <input
            class="emotion-4"
            min="0"
            placeholder="Введите"
            type="number"
            value=""
          />
          <span
            class="emotion-5"
            color="black"
            font-family="primary"
            font-weight="medium"
          >
            м2
          </span>
        </div>
      </div>
    </div>
    <div
      class="emotion-9"
      display="flex"
    />
    <div
      class="emotion-10"
    />
    <div
      class="emotion-11"
      display="flex"
    />
    <div
      class="emotion-8"
      display="flex"
      width="100%"
    >
      <div
        class="emotion-2"
        display="flex"
      >
        <span
          class="emotion-1"
          font-family="primary"
          font-size="normal"
          font-weight="medium"
        >
          Тип напольного покрытия?
        </span>
      </div>
      <div
        class="emotion-3"
        display="flex"
      />
      <div
        class="emotion-2"
        display="flex"
      >
        <div
          class="emotion-24"
          display="flex"
          width="100%"
        >
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Линолеум
              </span>
            </button>
          </div>
          <div
            class="emotion-19"
            display="flex"
          />
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Ламинат
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="emotion-11"
        display="flex"
      />
      <div
        class="emotion-2"
        display="flex"
      >
        <div
          class="emotion-24"
          display="flex"
          width="100%"
        >
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Паркет
              </span>
            </button>
          </div>
          <div
            class="emotion-19"
            display="flex"
          />
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Ковролин
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="emotion-11"
        display="flex"
      />
      <div
        class="emotion-2"
        display="flex"
      >
        <div
          class="emotion-24"
          display="flex"
          width="100%"
        >
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Половая доска
              </span>
            </button>
          </div>
          <div
            class="emotion-19"
            display="flex"
          />
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Керамическая плитка
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="emotion-11"
        display="flex"
      />
      <div
        class="emotion-2"
        display="flex"
      >
        <div
          class="emotion-24"
          display="flex"
          width="100%"
        >
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Пробковое покрытие
              </span>
            </button>
          </div>
          <div
            class="emotion-19"
            display="flex"
          />
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Мармолеум
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="emotion-11"
        display="flex"
      />
      <div
        class="emotion-2"
        display="flex"
      >
        <div
          class="emotion-24"
          display="flex"
          width="100%"
        >
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Полимерный пол
              </span>
            </button>
          </div>
          <div
            class="emotion-19"
            display="flex"
          />
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Другое
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="emotion-9"
      display="flex"
    />
    <div
      class="emotion-10"
    />
    <div
      class="emotion-11"
      display="flex"
    />
    <div
      class="emotion-8"
      display="flex"
      width="100%"
    >
      <div
        class="emotion-2"
        display="flex"
      >
        <span
          class="emotion-1"
          font-family="primary"
          font-size="normal"
          font-weight="medium"
        >
          Подготовка основания:
        </span>
      </div>
      <div
        class="emotion-3"
        display="flex"
      />
      <div
        class="emotion-2"
        display="flex"
      >
        <div
          class="emotion-24"
          display="flex"
          width="100%"
        >
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Подготовить стяжку пола
              </span>
            </button>
          </div>
          <div
            class="emotion-19"
            display="flex"
          />
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Подготовить бетонный пол
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="emotion-11"
        display="flex"
      />
      <div
        class="emotion-2"
        display="flex"
      >
        <div
          class="emotion-24"
          display="flex"
          width="100%"
        >
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Подготовить деревянный пол
              </span>
            </button>
          </div>
          <div
            class="emotion-19"
            display="flex"
          />
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Не требуется
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="emotion-11"
        display="flex"
      />
      <div
        class="emotion-2"
        display="flex"
      >
        <div
          class="emotion-24"
          display="flex"
          width="100%"
        >
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Другое
              </span>
            </button>
          </div>
          <div
            class="emotion-109"
            display="flex"
          />
          <div
            class="emotion-110"
            display="flex"
          />
        </div>
      </div>
    </div>
    <div
      class="emotion-9"
      display="flex"
    />
    <div
      class="emotion-10"
    />
    <div
      class="emotion-11"
      display="flex"
    />
    <div
      class="emotion-8"
      display="flex"
      width="100%"
    >
      <div
        class="emotion-2"
        display="flex"
      >
        <span
          class="emotion-1"
          font-family="primary"
          font-size="normal"
          font-weight="medium"
        >
          Дополнительные работы:
        </span>
      </div>
      <div
        class="emotion-3"
        display="flex"
      />
      <div
        class="emotion-2"
        display="flex"
      >
        <div
          class="emotion-24"
          display="flex"
          width="100%"
        >
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Демонтаж напольного покрытия
              </span>
            </button>
          </div>
          <div
            class="emotion-19"
            display="flex"
          />
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Монтаж напольных плинтусов
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="emotion-11"
        display="flex"
      />
      <div
        class="emotion-2"
        display="flex"
      >
        <div
          class="emotion-24"
          display="flex"
          width="100%"
        >
          <div
            class="emotion-18"
            display="flex"
          >
            <button
              class="emotion-17"
            >
              <span
                class="emotion-15"
              >
                <svg
                  color="#999999"
                  fill="none"
                  height="18"
                  stroke="#999999"
                  stroke-linecap="butt"
                  stroke-linejoin="miter"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polyline
                    points="20 6 9 17 4 12"
                  />
                </svg>
              </span>
              <span
                class="emotion-16"
                color="mountainmist"
                font-family="primary"
                font-size="tiny"
                font-weight="semiBold"
              >
                Монтаж дверных порогов
              </span>
            </button>
          </div>
          <div
            class="emotion-109"
            display="flex"
          />
          <div
            class="emotion-110"
            display="flex"
          />
        </div>
      </div>
    </div>
    <div
      class="emotion-9"
      display="flex"
    />
    <div
      class="emotion-10"
    />
  </div>
</DocumentFragment>
`;
